webservice/src/main/java/com/hortonworks/registries/webservice/RegistryConfiguration.java

```diff
@@ -17,10 +17,7 @@
 package com.hortonworks.registries.webservice;
```

Comment:
"authorization" configuration does not seem to have been added. However, you seem to be using it in AuthorizationAgentFactory class.

Reply:
Authorization is configured by the authorization-agent module. It is possible to configure this in the config file.